src/unlimited.cpp

```diff
@@ -1212,23 +1213,27 @@ bool CheckAndRequestExpeditedBlocks(CNode* pfrom)
     {
       BOOST_FOREACH(string& strAddr, mapMultiArgs["-expeditedblock"]) 
         {
-          // Add the peer's listening port if it is empty
-          int pos1 = strAddr.rfind(":");
-          int pos2 = strAddr.rfind("]:");
-          if (pos1 <= 0 && pos2 <= 0)
-              strAddr += ':' + boost::lexical_cast<std::string>(pfrom->addrFromPort);
-
```

Comment:
This whole code was moved around rather than deleted. It makes it very hard to figure out what actually changed and what is just shuffling the code around. Refactoring and logic belong to separate PRs.